spec/integration/queue_operations/queue_operations_spec.rb

```
@@ -0,0 +1,84 @@
+assert(received.first.is_a?(Shoryuken::Message), "Received items should be Message objects")
+
+# Test 9: Batch message deletion
+entries = received.map.with_index do |msg, idx|
+  { id: idx.to_s, receipt_handle: msg.receipt_handle }
+end
+delete_result = queue.delete_messages(entries: entries)
+refute(delete_result, "Delete should succeed without failures")
+
+# Test 10: FIFO queue message sending with auto-generated attributes
+fifo_send_result = fifo_queue.send_message(message_body: 'fifo test message')
+assert(fifo_send_result.message_id, "FIFO send should have message_id")
```

RuboCop (1.81.7)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)